src/_sass/home.scss

```diff
@@ -92,6 +92,15 @@
     * {
         color: #FFF;
     }
+    .home--button {
+        background-color: #FFF;
```

Comment:
Don't we have a var for this color? if not, we should create it

Reply:
Hm, are you asking for vars for "white-lh" and "black-lh"? I can do this in another PR